modules/persistent-storage-csi-smb-cifs-create-pv.adoc

```diff
@@ -0,0 +1,192 @@
+<3> `volumeHandle` format: {smb-server-address}#{sub-dir-name}#{share-name}. Ensure that this value is unique for every share in the cluster.
+<4> The Samba server must be installed somewhere and reachable from the cluster with <hostname> being the hostname for the Samba server and <shares> the path the server is configured to have among the exported shares.
+<5> The name of the Secret, which is referenced in the storage class (see _Prerequisites_).
+<6> The applicable namespace.
+
+. Create a PVC:
+
+.. Create a PVC by running the following command with the following example YAML file:
++
+[source, cli]
+----
+oc create -f <pv-file-name>.yaml <1>
+----
+<1> The name of the PVC YAML file.
++
+.Example PVC YAML file
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc-name> <1>
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: <storage-amount> <2>
+  volumeName: <pv-name> <3>
+  storageClassName: <sc-name> <4>
+----
+<1> The name of the PVC.
+<2> Storage request amount.
+<3> The name of the PV from the first step.
+<4> The name of the CIFS/SMB storage class that you created earlier (see _Prerequisites_).
+
+.. Ensure that the PVC was created and is in the "Bound" status by running the following command:
++
+[source, cli]
+----
+oc describe pvc <pvc-name> <1>
+----
+<1> The name of the PVC name that you created in the preceding step.
++
+.Example output
++
+[source,terminal]
+----
+Name:          pvc-test
+Namespace:     default
+StorageClass:  smb
+Status:        Bound <1>
+...
+----
+<1> PVC is in Bound status.
+
+. Create a deployment on Linux by running the following command with following example YAML file:
++
+[source, cli]
+----
+oc create -f <deployment-file-name>.yaml <1>
+----
+<1> The name of the deployment YAML file.
++
+.Example deployment YAML file
++
+[source, yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+  name: <deployment-name> <1>
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+      name: <deployment-name> <1>
+    spec:
+      nodeSelector:
+        "kubernetes.io/os": linux
+      containers:
+        - name: <deployment-name> <1>
+          image: quay.io/centos/centos:stream8
+          command:
+            - "/bin/bash"
+            - "-c"
+            - set -euo pipefail; while true; do echo $(date) >> <mount-path>/outfile; sleep 1; done <2>
+          volumeMounts:
+            - name: <vol-mount-name> <3>
+              mountPath: <mount-path> <2>
+              readOnly: false
+      volumes:
+        - name: <vol-mount-name> <3>
+          persistentVolumeClaim:
+            claimName: <pvc-name> <4>
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+----
+<1> The name of the deployment.
+<2> The volume mount path.
```

Comment:
Nit - since this callout describes a value that's both on its own and in the middle of a sort of command, perhaps you can reword to add "specify"? That way this label avoids potentially describing that long command as the mount path:
```suggestion
<2> Specify the volume mount path.
```

Up to you though